src/index.html

```diff
@@ -11,6 +11,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     action="http://localhost:3000/api"
+     method="post"
+    >
+      <label class="big-search">
+        <div
+          class="search-bar"
+          data-qa="big"
+        >
+          <img
+            src="./images/Search.svg"
+            alt="123"
+            class="image"
+          >
+          <input
+            class="input"
+            name="default-search-bar"
+            type="text"
+            placeholder="Try &quot;Los Angeles&quot;"
+          >
+        </div>
+
+      </label>
```

Comment:
```suggestion
        </div>
      </label>
```
Don't leave empty lines between parent and child elements. Also, IMHO, div is redundant here, as you already have label as a container